GAE/src/org/akvo/flow/api/app/DataPointUtil.java

```diff
@@ -35,10 +33,18 @@
 import org.waterforpeople.mapping.domain.SurveyInstance;
 import org.waterforpeople.mapping.serialization.response.MediaResponse;
 
-import com.gallatinsystems.survey.dao.QuestionDao;
-import com.gallatinsystems.survey.domain.Question;
-import com.gallatinsystems.surveyal.domain.SurveyedLocale;
 import javax.annotation.Nullable;
+import java.util.ArrayList;
```

Comment:
There are lots of changes to import *style* and *order* that are not directly to the change business logic change we want.

Reply:
the IDE does this, we need a common code style for all IDEs so this does not happen

Reply:
it does it with you reformat the whole file. We *had* a common style defined for Eclipse but not for IntelliJ IDEA